Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": {
+    "text": "SQL input binding",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "3000"
```

Comment:
I wasn't sure where these ids come from so I just put these random numbers for now.